butterfree/clients/cassandra_client.py

```diff
@@ -42,12 +42,12 @@ class CassandraClient(AbstractClient):
     def __init__(
         self,
         cassandra_host: List[str],
-        cassandra_key_space: str,
+        cassandra_keyspace: str,
```

Comment:
Just asking myself why do we have this prefix "cassandra" for all the attributes if this is a CassandraClient LMAO

Reply:
I don't know, we can change this :rofl: